v1.1.yml

```diff
@@ -1,6 +1,6 @@
 swagger: '2.0'
 info:
-  title: clever-api
+  title: Data API
```

Comment:
Events should be it's own API (and we should probably move all the `/resource/{id}/events` endpoints there too).

Does that mess anything up (e.g. client libraries) to separate out events?